shared/derive/src/replicate.rs

```diff
@@ -426,12 +526,24 @@ pub fn new_complete_method(
         fields = new_output_result;
     }
 
-    quote! {
-        pub fn new_complete(#args) -> #replica_name {
+    let fn_inner = if is_replica_tuple_struct {
+        quote! {
+            #replica_name (
```

Comment:
There might be a better way to handle this `(` vs `{` issue for named vs unnamed structs...

Reply:
I think it looks great :+1: